docs/running-a-node/docker-management.md

```diff
@@ -144,6 +144,27 @@ Once modifications are complete, [start up the docker container again](#starting
 
 ---
 
+### Docker Compose
+
+A sample docker-compose.yml is provided to model the same behavior as the docker cli examples above
+
+```yml
+version: '3'
+services:
+  node:
+    image: "nanocurrency/nano:${NANO_TAG}" # tag you wish to pull, none for latest
+    restart: "unless-stopped"
+    ports:
+     - "7075:7075/udp"   #udp network traffic
```

Comment:
No need for UDP anymore, maybe should remove it from elsewhere (another PR)

Reply:
yeah the goal of this PR was to make an analog of the existing CLI commands for docker-compose, I would be fine doing it in this PR but I feel if we want to remove UDP it would be best to do in a single PR with that intent.